### PR TITLE
Use the built-in laser pointer for magicleap

### PR DESCRIPTION
**Pull request by asajeffrey**
Use the built-in laser pointer on magicleap, now that there is one.

---

- [X] `./mach build -d` does not report any errors
- [X] `./mach test-tidy` does not report any errors
- [X] These changes do not require tests because we don't test native UI elements

---
This change is [Reviewable](https://reviewable.io/reviews/servo/servo/23953)

### CODE REVIEW

File: support/magicleap/Servo2D/scenes/Servo2D.design [hunk 1]
```diff
@@ -4,6 +4,7 @@
   <property id="sceneName" value="Servo2D"/>
   <node name="contentPanel" nodeTypeId="lumin.ui.panel">
     <property id="cursorInitialPosition"/>
+    <property id="cursorVisible" value="false"/>
```
Comment (Manishearth) [inline support/magicleap/Servo2D/scenes/Servo2D.design]: is this what makes the builtin laser visible? how does setting it to false work?

**Review by Manishearth — APPROVED** [review]
looks good, bit unsure about which code actually turns on the builtin laser

File: support/magicleap/Servo2D/scenes/Servo2D.design (hunk 1 above)
Comment (asajeffrey) [reply support/magicleap/Servo2D/scenes/Servo2D.design]: https://mozilla.logbot.info/servo/20190812#c16536817

**Comment by asajeffrey** [comment]
```
Internal Server Error, incidentId 7d20a4a9-7177-4f29-aa15-df3f499ad4d4.
```
@bors-servo retry

**Pull request merged**